App/Model/Collection.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Model;
+
+
+use Iterator;
+
+class Collection implements Iterator
+{
+
+
+    public function __construct(protected array $items)
+    {
+    }
+
+    public
```

Comment:
Why is this on the line above ? 

it can just be public function current() etc

Reply:
error in formatting the IDE formatted it this way … didn't realize it